lib/pkgcloud/azure-v2/compute/client/servers.js

```diff
@@ -40,19 +41,17 @@ function getLimits(callback) {
 function getServers(callback) {
   var self = this;
 
-  self.login(function (err) {
+  self.login(err => {
```

Comment:
This format is not supported in the "declared" version of this repo (node 0.10 or something) so it has to be function (err) {...}

[Here and everywhere]